weave/execute_fast.py

```diff
@@ -112,6 +112,7 @@ def _can_fast_map(map_fn):
         lambda n: isinstance(n, graph.OutputNode)
         and (
             op_can_be_async(n.from_op.name)
+            or "://" in n.from_op.name
```

Comment:
nit: would be nice to wrap this in a function `op_is_likely_loaded` or something

Reply:
Yes, I will do a follow up PR after the two are merged.